### PR TITLE
Load certificates from Windows-ROOT cert store

### PR DESCRIPTION
**Pull request by flippingbits**
## Summary of Changes

On Windows, load certificates from the cert store `WINDOWS-ROOT` in addition to `WINDOWS-MY`.

Also, update the level of a few log statements so that we can better troubleshoot situations where users have difficulty loading certificates.

## Pull request checklist

Please check if your PR fulfills the following (if applicable):

- Tests:
    - [ ] Added new
    - [ ] Updated existing
    - [ ] Deleted existing
- [ ] Have you validated this change locally against a running instance of the Quarkus dev server?
    ```shell
    make quarkus-dev
    ```
- [ ] Have you validated this change against a locally running native executable?
    ```shell
    make mvn-package-native && ./target/ide-sidecar-*-runner
    ```

### CODE REVIEW

File: src/main/java/io/confluent/idesidecar/restapi/util/WebClientFactory.java [hunk 1]
```diff
@@ -163,14 +166,14 @@ void addCertsFromBuiltInTrustStore(PemTrustOptions pemTrustOptions) {
...
-      Log.debug("Loading certificates from system key store.");
+      Log.infof("Loading certificates from system key store %s.", keyStoreType);
       var keyStore = KeyStore.getInstance(keyStoreType);
       keyStore.load(null, null);
       var it = keyStore.aliases().asIterator();
       while (it.hasNext()) {
         var certAlias = it.next();
         var cert = keyStore.getCertificate(certAlias);
-        Log.debugf(
+        Log.infof(
```
Comment (rhauch) [inline src/main/java/io/confluent/idesidecar/restapi/util/WebClientFactory.java:176]: How many log lines will this generate, typically? A few or many?

**Review by rhauch — COMMENTED** [review]
I think this change makes sense and I'm happy to approve, pending successful testing on Windows. I do have one question about the number of `Adding certificate ...` info log messages we might typically see. A handful or even a dozen is fine, but more than a few dozen might be problematic.

**Comment by flippingbits** [comment]
> I do have one question about the number of Adding certificate ... info log messages we might typically see. A handful or even a dozen is fine, but more than a few dozen might be problematic.

Good point -- thanks for raising the concern! Note that these log messages are needed to debug loading certs from the system cert stores on Windows. We can probably revert the changed log level in the next extension release or the release after the next one, once loading certs on Windows is more stable. The messages are only logged when initializing the `WebClient`, which happens once at startup and every time the user updates the preferences. I'd expect a few dozen certs/log messages but it depends on the number of certs maintained in the system cert store.

**Review by rhauch — APPROVED** [review]
LGTM. This still adds a few more info-level log messages, and only outputs the _number_ of certificates added.

**Review by shouples — APPROVED** [review]
Works great on Windows!

**Pull request merged**